website/docs/language/functions/setsubtract.html.md

```diff
@@ -10,7 +10,7 @@ description: |-
 # `setsubtract` Function
 
 The `setsubtract` function returns a new set containing the elements from the first set that are not present in the second set. In other words, it computes the
-[relative complement](https://en.wikipedia.org/wiki/Complement_(set_theory)#Relative_complement) of the first set in the second set.
+[relative complement](https://en.wikipedia.org/wiki/Complement_(set_theory)#Relative_complement) of the second set in the first set.
```

Comment:
Thank you for spotting this! I took a look and I think we can push even more with this edit. The relative complement of the second set is the elements from the first set that aren't in the second set. 

```suggestion
[relative complement](https://en.wikipedia.org/wiki/Complement_(set_theory)#Relative_complement) of the second set.
```